packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -2502,6 +2502,29 @@ describe('<Autocomplete />', () => {
     });
   });
 
+  it('should not override internal listbox ref when external listbox ref is provided', () => {
```

Comment:
Previoulsy When external listbox ref is provided, internal ref was overriden by it. In turn [this](https://github.com/mui/material-ui/blob/master/packages/mui-base/src/useAutocomplete/useAutocomplete.js#L572) function would not get called which makes [syncHighlightedIndex](https://github.com/mui/material-ui/blob/master/packages/mui-base/src/useAutocomplete/useAutocomplete.js#L579) not get called